charts/brigade/values.yaml

```diff
@@ -125,6 +125,35 @@ apiserver:
     type: NodePort
     # nodePort: 31600
 
+scheduler:
+
+  image:
+    repository: krancour/brigade-scheduler
```

Comment:
I assume this is a temporary image pointing to your registry?

Reply:
For some reasons I recall having fixed this throughout `values.yaml` recently. Looking into that. Maybe that was in another branch.

In practice, it's only a placeholder at the moment. `make hack` builds and deploys the whole shooting match and it will override that value with the name of whatever image was just built.

This will become more relevant when we start actually building and releasing the chart (even as preview).

Reply:
Ok... I wasn't imagining it. I have a s/krancour/brigadecore/ queued up in one of my other PRs.

Reply:
Sure, this can be addressed later, just wanted to make sure we don't forget about it.